### DISCUSSION TITLE
validação de nota  NFC-e

### DISCUSSION THREAD

**Discussion by gilsonlima10** [OCA/l10n-brazil#3291] [Q&A]
ao tentar validar uma NFC no backend tras a seguinte mensagem:
Erros de validação XML:Element '{http://www.portalfiscal.inf.br/nfe}xNome': This element is not expected. Expected is one of ( {http://www.portalfiscal.inf.br/nfe}CNPJ, {http://www.portalfiscal.inf.br/nfe}CPF, {http://www.portalfiscal.inf.br/nfe}idEstrangeiro ).
Element '{http://www.portalfiscal.inf.br/nfe}xNome': This element is not expected. Expected is one of ( {http://www.portalfiscal.inf.br/nfe}CNPJ, {http://www.portalfiscal.inf.br/nfe}CPF ).

**Comment by antoniospneto** [comment]
Veja o que a mensagem está te falando,  está faltando a informação do CNPJ ou CPF ou ID Estrangeiro no XML da nota.
Essa informação deveria vir preenchida antes do Nome.

Confira se nos cadastros do Parceiro ou dá Empresa essa informação está preenchida corretamente, penso que deve estar faltando..
